src/CVPresenter.ts

```diff
@@ -34,21 +34,21 @@ export function skillAsText (store: Store, sk: Node):string {
   }
   const manual = store.anyJS(sk as NamedNode, ns.vcard('role'))
   if (manual) return manual
-  return '¿¿¿ skill ???'
+  return ''
```

Comment:
leave this in as it was with the '¿¿¿ skill ???' because this is a special case and we need to know when it occurs. We will fix it next time.